src/main/java/org/squiddev/cobalt/LuaThread.java

```diff
@@ -459,6 +461,9 @@ public void run() {
 						// Clear the function after the first run
 						LuaFunction function = func;
 						func = null;
+						if (function instanceof LuaInterpretedFunction) {
```

Comment:
Not really sure what we want to do about environments in coroutines.

I'm honestly tempted just to remove them. Have `getfenv` return `_G` and `setfenv` do nothing. Yes, it'll break some of the tests, but I don't _think_ it'll affect any actual Lua programs.